modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlCommandHandlerExceptionHandlingTest.java

```diff
@@ -67,6 +71,7 @@ void before() {
         HybridClock clock = new HybridClockImpl();
         catalogManager = createTestCatalogManager("test", clock);
         assertThat(catalogManager.startAsync(new ComponentContext()), willCompleteSuccessfully());
+        awaitDefaultZoneCreation(catalogManager);
```

Comment:
we had flaky here

Reply:
as far as I see, this flakiness highlights the real issue, and you decided to just obscure it with this wait.

Initial version of Catalog uses `0L` as activation time, which is not valid value according to HybridTimestamp' constructor:

```
    private HybridTimestamp(long time) {
        this.time = time;

        // Negative time breaks comparison, we don't allow overflow of the physical time.
        // "0" is a reserved value for "NULL_HYBRID_TIMESTAMP".
        if (time <= 0) {
            throw new IllegalArgumentException("Time is out of bounds: " + time);
        }
    }
```

Reply:
revert the change and mute ander the https://issues.apache.org/jira/browse/IGNITE-24339

Reply:
fix reverted back and the test muted under ticket.